atd-vze/src/views/Crashes/Maps/Pin.js

```diff
@@ -1,27 +1,25 @@
-import React, { PureComponent } from "react";
+import React from "react";
 import styled, { keyframes, css } from "styled-components";
 import { colors } from "../../../styles/colors";
 import { pinStyles, PIN_ICON } from "../../../styles/mapPinStyles";
-export default class Pin extends PureComponent {
-  render() {
-    const {
-      color = "warning",
-      size = 40,
-      isDragging,
-      animated = false,
-    } = this.props;
 
-    const pinStyle = {
-      fill: colors[color],
-      stroke: pinStyles["stroke"],
-      strokeWidth: pinStyles["strokeWidth"],
-      // Move pin up on drag and down when dropped
-      transform: `translate(${-size / 2}px, ${
-        isDragging && animated ? `-65px` : `${-size}px`
-      })`,
-    };
+const Pin = ({
+  color = "warning",
+  size = 40,
+  isDragging = false,
+  animated = false,
+}) => {
+  const pinStyle = {
+    fill: colors[color],
+    stroke: pinStyles["stroke"],
+    strokeWidth: pinStyles["strokeWidth"],
+    // Move pin up on drag and down when dropped
+    transform: `translate(0px, ${
+      isDragging && animated ? `-${size}px` : `${-size / 2}px`
+    })`,
```

Comment:
I had to update this style because the map was rendering the pin marker way off center with the old transform/translate values.